latex/pdfpc/pdfpc.sty

```diff
@@ -150,7 +150,11 @@ ______</rdf:Description>^^J%
 \else%
   \newcommand{\pdfpcnote}[1]{%
     {%
-      \edef\\{\string\n}%
+      \ifPDFPC@disablemarkdown%
```

Comment:
Hmm no, the idea was to pass *real* linebreaks to MD. Like

```
\pdfpcnote{
| Tables        | Are           | Cool  |\\
| ------------- |:-------------:| -----:|\\
| col 3 is      | right-aligned | $1600 |\\
| col 2 is      | centered      |   $12 |\\
| zebra stripes | are neat      |    $1 |\\
}
```

Reply:
Ah, I see. I will reset the branch then.